package.json

```diff
@@ -30,7 +30,8 @@
     "postinstall": "./scripts/lambda-functions-dependencies.sh && npm run build-lambdas",
     "watch-lambdas": "./scripts/watch-lambdas.js",
     "build-lambdas": "npm-run-all \"amplify:*\"",
-    "amplify:sendNotification": "./scripts/bundle-lambda.sh \"$npm_lifecycle_event\""
+    "amplify:sendNotification": "./scripts/bundle-lambda.sh \"$npm_lifecycle_event\"",
+    "amplify:subscribeToColonyPushNotifications": "./scripts/bundle-lambda.sh \"$npm_lifecycle_event\""
```

Comment:
Are these scripts required for all lambdas that we convert to typescript? Or just the ones that deal with notifications ?

Reply:
Unfortunately, we will need to do this for every lambda that we convert to typescript, as it is how amplify knows how to build these lambdas before sending them to AWS lambda on the `amplify push` command. 

It's a bit of a drawback to have these clutter the package.json, but I think having typescript in the lambdas is worth the trade-off